vm-setup/roles/firewall/tasks/iptables.yaml

```diff
@@ -36,19 +38,29 @@
 
 - name: "iptables: Ironic Ports"
   iptables:
-    chain: FORWARD
-    in_interface: "{{ provisioning_interface }}"
+    chain: "{{ item[0].chain }}"
+    action: insert
+    in_interface: "{{ item[0].interface }}"
     protocol: tcp
     match: tcp
-    destination_port: "{{ item }}"
+    destination_port: "{{ item[1] }}"
     jump: ACCEPT
     state: "{{ firewall_rule_state }}"
-  loop: "{{ ironic_ports }}"
+  vars:
+    interfaces:
+      - chain: INPUT
+        interface: "{{ provisioning_interface }}"
+      - chain: INPUT
+        interface: "{{ cluster_provisioning_interface }}"
```

Comment:
what is difference between `provisioning_interface` and `cluster_provisioning_interface`

Reply:
the provisioning_interface is a direct reference to the provisioning network, while the cluster_provisioning_interface refers to the ironic endpoint